.github/workflows/wheels.yml

```diff
@@ -28,11 +28,11 @@ jobs:
         import itertools
         environ = os.environ
 
-        PY_VERSIONS = ['3.9', '3.10', '3.11', '3.12']
+        PY_VERSIONS = ['3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
I see that indeed PyTorch 2.6.0 will support Python 3.13. Did they drop any old versions?

Reply:
They drop 3.8